charaData.json

```diff
@@ -5411,6 +5411,50 @@
         "imageURL": "./charaimgs/3040000000_03.png",
         "en": "Gawain"
     },
+    "ソシエ(風属性ver)": {
+        "name": "ソシエ(風属性ver)",
+        "ja": "ソシエ(風属性ver)",
+        "element": "wind",
+        "type": "heal",
+        "race": "erune",
+        "sex": "female",
+        "fav1": "fist",
+        "fav2": "music",
+        "support": "none",
+        "support2": "none",
+        "support3": "none",
+        "minhp": "300",
+        "hp": "1590",
+        "minattack": "1080",
+        "attack": "5760",
+        "baseDA": 7.0,
+        "baseTA": 3.0,
+        "ougiRatio": 4.5,
+        "imageURL": "./charaimgs/3040241000_01.png",
+        "en": "Societte (Wind)"
+    },
+    "コウ": {
+        "name": "コウ",
+        "ja": "コウ",
+        "element": "dark",
+        "type": "heal",
+        "race": "erune",
+        "sex": "male",
+        "fav1": "katana",
+        "fav2": "music",
+        "support": "none",
+        "support2": "none",
+        "support3": "none",
+        "minhp": "240",
+        "hp": "1389",
+        "minattack": "1389",
```

Comment:
> XXX: check original data source, min ATK is not wrong paste?

No.

image from internet
![image](https://user-images.githubusercontent.com/22173829/69368041-b22f3c80-0cdc-11ea-91a5-ed75bfc10179.png)

Reply:
thx, ok I just doubt it coz that was same with maxHP